### DISCUSSION TITLE
About inferring ancestral allele states

### DISCUSSION THREAD

**Discussion by bguo068** [tskit-dev/tsinfer#637] [Q&A]
Hi everyone, 

Sorry for the long questions. I am trying to prepare data for running `tsinfer` ([Kelleher et al 2019](https://www.nature.com/articles/s41588-019-0483-y)) on some malaria parasite WGS data, but got stuck with details in inferring the ancestral states and filtering the results. I learned from the tsinfer paper and Jana’s talk that we could use `est-sfs` ([Keightley and Jackson, 2018](https://academic.oup.com/genetics/article/209/3/897/5930981?login=false)) to estimate the ancestral allele state. It seems not very hard to run the program itself. However, it is not clear to me: 
- **how to choose the outgroup species**: Keightley’s paper says that if the outgroup is too close to the focal species then it violates their assumption that all focal species alleles coalesce within the first branch. (“We used orangutan and macaque as outgroups in our analysis. Chimpanzee and gorilla are closer and potentially more informative, but they share a high proportion of polymorphism with human and this violates an assumption of our analysis.“)   Are there any suggestions/comments on how we should choose the outgroup species that do not violate the assumption?
- **how to align** WGS data or reference genomes of different outgroup species to a reference genome of interest in order prepare input data file for `est-sfs`: Keightley’s paper used EPO to align 6 primates. I do not know if there is a similar tool for non-human species. Can I just try to find multiple WGS raws reads for the focal species and one high-quality WGS reads for each outgroup species, and align them to the reference genome of the focal species using bowtie2 or bwa? Or should I just use some multi-aligner like GSalign (by [Lin et al 2020](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6569-1))  to directly align the outgroup species’ reference gnomes to the reference gnome of the focal species?
- **how to filter** the `est-sfs` results (i.e what threshold do we use to say we are confident about the inferred ancestral allele). `est-sfs` generates a result file contains the probability of the major allele being ancestral for each site. It ranges from 0 to 1. We will be confident about the ancestral state if the probability is close 0 or 1 but hard to determine if the probability is in the middle. Do we have any suggestions on what threshold should be used to filter the sites based on the probability of major allele being ancestral?

Any suggestions or comments would be appreciated! Thank you so much!

**Comment by bguo068** [comment]
From Jana Obšteter (pasted from a slack thread)

>hi! So, I can only tell you what I did - but this doesn't mean its right :sweat_smile:
>- My understanding about the outgroups is that if for example you are examining a species, then your outgroups can be species from the same genus (or possibly even family). If you are interested in a genus, then your outgroups can be other genus in the same family ... At least that is what we did
>- The multiple genome alignment is quite a project on its own. There are two whole genome aligners out there (that I've used): Mauve (I think this is reference based) and Cactus (doesn't need a reference to align to). However, there might be some alignments or EPO (Enredo, Pecan and Ortheus) already ready for you. You can check on Ensembl and [Ines Rebollo] can tell you a bit more about it since she found the EPO for her species.
>- I don't know the answer to this question - I just took them all. But you can contact the authors directly. If you do, please let me know

**Comment by hyanwong** [comment]
There's a thread at https://github.com/tskit-dev/tsinfer/discussions/523 too
